src/Settings.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace JsonI18n;
+
+/**
+ * Internationalization class to handle date formatting
+ * @author Vincent Diep
+ * @license https://www.mozilla.org/en-US/MPL/2.0/ MPL-2.0
+ */
+class Settings
+{
+    /**
+     * The type of fallback
+     * @var bool
+     */
+    protected $fallbackWithKey;
```

Comment:
I am thinking that maybe the option should be called `strict`, similar to how it is called `strict_variables` for Twig with a similar behaviour: https://twig.symfony.com/doc/2.x/api.html#environment-options

Reply:
Indeed, `strict` is a better name! :)